1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+---
+post_title: Quick Start
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+
+# Launch a long running interactive Bash session
+
+In this example, a long running job app is launched and then the `dcos exec` command is used to launch an interactive Bash session inside of the task container on the node.
+
+1.  Deploy and run a job app with the DC/OS CLI:
+
+    1.  Create the following app definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+          "id": "task-exec-test",
+          "labels": {},
+          "run": {
+            "artifacts": [],
+            "cmd": "sleep 100000000",
+            "cpus": 0.01,
+            "disk": 0,
+            "env": {},
+            "maxLaunchDelay": 3600,
+            "mem": 32,
+            "placement": {
+              "constraints": []
+            },
+            "restart": {
+              "policy": "NEVER"
+            },
+            "volumes": []
+          }
+        }
+        ```
+    
+    1.  Deploy the job app with this CLI command:
+    
+        ```bash
+        $ dcos job add test-job.json
+        ```
+        
+    1.  Verify that the app has been successfully deployed:
+    
+        ```bash
+        $ dcos job list
+        ID              DESCRIPTION  STATUS   LAST SUCCESFUL RUN        
+        task-exec-test               Unscheduled         None
+        ```
+
+    1.  Run the job:
+    
+        ```bash
+        $ dcos job run task-exec-test
+        ```
+
+1.  Get the task ID of the job with this CLI command:
```

Comment:
The markdown processor figures out the actual order during the build - that way we don't manually keep the numbers in sync.